doc/src/cylc-user-guide/cug.tex

```diff
@@ -4379,6 +4379,7 @@ \subsubsection{Parameter Expansion}
     [[parameters]]
         obs = ship, buoy, plane
         run = 1..5  # 1, 2, 3, 4, 5
+        idx = 1..9^2  # 1, 3, 5, 7, 9
```

Comment:
+1 for `START..END..STEP`. In python's slice syntax the end-point is excluded i.e. `1:9:2 = 1, 3, 5, 7`, whereas in bash this is inclusive i.e. `1..9..2 = 1, 3, 5, 7, 9`.